packages/twenty-front/src/modules/object-record/record-table/constants/TableZIndex.ts

```diff
@@ -6,7 +6,7 @@ export const TABLE_Z_INDEX = {
     editMode: 20,
   },
   footer: {
-    default: 12,
+    default: 18,
```

Comment:
I don't think it makes really sense as Z-index are locale (there is no point comparing footer to cell sticky for instance

Reply:
Already discussed on Discord.